src/vs/monaco.d.ts

```diff
@@ -3808,14 +3808,22 @@ declare namespace monaco.editor {
 		enabled?: boolean;
 	}
 
-	/**
-	 * Configuration options for editor sticky scroll
-	 */
-	export interface IEditorStickyScrollOptions {
+	export interface IEditorExperimentalOptions {
 		/**
-		 * Enable the sticky scroll
+		 * Configuration options for editor sticky scroll
 		 */
-		enabled?: boolean;
+		stickyScroll?: {
+			/**
+			 * Enable the sticky scroll
+			 */
+			enabled?: boolean;
+		};
+	}
+
+	export interface EditorExperimentalOptions {
```

Comment:
It looks like there are more editor experimental settings, though.

![Screenshot showing other experimental editor settings in the Settings UI](https://user-images.githubusercontent.com/7199958/181829984-21a8274e-f28a-4b2c-8fb0-c8ae44110651.PNG)